website/source/docs/providers/aws/d/iam_policy_document.html.markdown

```diff
@@ -71,14 +72,15 @@ valid to use literal JSON strings within your configuration, or to use the
 
 The following arguments are supported:
 
-* `id` (Optional) - An ID for the policy document.
+* `version` (Optional) - An policy version. Defaults to `2012-10-17`.
```

Comment:
We should probably document that if you set this to `2008-10-17` then you can't use policy variables.

(Though I don't know why anyone would set this to anything other than what it defaults to anyway.)